lib/presentation/molecules/tip_resource_molecule.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:infinite_horizons/presentation/atoms/separator_atom.dart';
+import 'package:infinite_horizons/presentation/molecules/top_bar_molecule.dart';
+
+class TipResourceMolecule extends StatelessWidget {
+  const TipResourceMolecule(
+      {required this.innerWidget, required this.title, super.key,
+      this.topBarType = TopBarType.back,
+      this.separatorVariant = SeparatorVariant.closeWidgets,});
```

Comment:
Please indent  the page

Reply:
Do you mean padding it?

Reply:
Running `dart format path1` so that the code will be organized.

Reply:
Ok